packages/resolve-command/test/index.tests.js

```diff
@@ -0,0 +1,153 @@
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+import createStore from '../../resolve-es/src';
+import memoryEsDriver from '../../resolve-es-memory/src';
+
+import createBus from '../../resolve-bus/src';
+import memoryBusDriver from '../../resolve-bus-memory/src';
+
+import commandHandler from '../src';
+
+describe('command', () => {
+    let store;
+    let bus;
+    let execute;
+    let aggregates;
+    let testCommand;
+    let testEvent;
+
+    beforeEach(() => {
+        testCommand = {
+            __aggregateId: 'test-id',
+            __aggregateName: 'USER',
+            __commandName: 'CREATE',
+            name: 'Vasiliy'
+        };
+
+        testEvent = {
+            __aggregateId: 'test-id',
+            __type: 'USER_CREATED',
+            name: 'Vasiliy'
+        };
+
+        aggregates = {
+            USER: {
+                CREATE: () => testEvent
+            }
+        };
+
+        store = createStore({ driver: memoryEsDriver() });
+        bus = createBus({ driver: memoryBusDriver() });
+        execute = commandHandler({ store, bus, aggregates });
+    });
+
+    it('should save and publish event', () => {
+        const eventHandlerSpy = sinon.spy();
+        bus.onEvent(['USER_CREATED'], eventHandlerSpy);
+
+        return execute(testCommand).then(() => {
+            expect(eventHandlerSpy.callCount).to.be.equal(1);
+            expect(eventHandlerSpy.lastCall.args[0])
+                .to.be.deep.equal(testEvent);
+
+            const storedEvents = [];
+            return store.loadEventsByAggregateId('test-id', event => storedEvents.push(event))
+                .then(() => expect(storedEvents)
+                    .to.be.deep.equal([testEvent])
+                );
+        });
+    });
+
+    it('should reject event in case of aggregateName absense', () => {
+        Reflect.deleteProperty(testCommand, '__aggregateName');
```

Comment:
Why not `delete testCommand.__aggregateName` or `testCommand.__aggregateName = null;`?